src/main/java/net/mcreator/ui/views/editor/image/tool/tools/AbstractModificationTool.java

```diff
@@ -27,9 +27,10 @@
 import java.awt.event.MouseEvent;
 
 public abstract class AbstractModificationTool extends AbstractTool {
+
 	public AbstractModificationTool(String name, String description, ImageIcon icon, Canvas canvas,
 			ColorSelector colorSelector, VersionManager versionManager) {
-		super(name, description, icon, canvas, colorSelector, versionManager);
+		super(name, description, icon, canvas, colorSelector, versionManager, "Opacity:");

```

Comment:
This is not the way to do this (explained better in other comments)

Reply:
:heavy_check_mark: Fixed